src/ai/backend/manager/api/wsproxy.py

```diff
@@ -202,22 +203,23 @@ async def upstream(self) -> None:
 
     async def downstream(self) -> None:
         try:
-            self.upstream_buffer_task = asyncio.create_task(
-                self.consume_upstream_buffer(),
-            )
-            async for msg in self.up_conn:
-                if msg.type == aiohttp.WSMsgType.TEXT:
-                    await self.down_conn.send_str(msg.data)
-                    if self.downstream_cb is not None:
-                        await asyncio.shield(self.downstream_cb(msg.data))
-                if msg.type == aiohttp.WSMsgType.BINARY:
-                    await self.down_conn.send_bytes(msg.data)
-                    if self.downstream_cb is not None:
-                        await asyncio.shield(self.downstream_cb(msg.data))
-                elif msg.type == aiohttp.WSMsgType.CLOSED:
-                    break
-                elif msg.type == aiohttp.WSMsgType.ERROR:
-                    break
+            with aiotools.PersistentTaskGroup() as tg:
```

Comment:
This line should use `async with`, not `with`.

Reply:
https://github.com/lablup/backend.ai-manager/pull/533/commits/5354d49467f183b191e68d0e8f7c9091e51c84c0 resolves this issue.